src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useConnectionsProvider } from '../../contexts/ConnectionsProvider';
+import { Box, Button } from '../base';
+import { Connection } from './Connection';
+import { SidebarGroup, SidebarTitle } from './styles';
+
+export function Sidebar() {
+  const navigate = useNavigate();
+
+  const { connections, selectConnection } = useConnectionsProvider();
+
+  const handleNewConnection = useCallback(() => {
+    navigate('/');
+    selectConnection(undefined);
+  }, [navigate]);
```

Comment:
does navigate should be a dependency?

Reply:
The useNavigate hook has some dependencies inside the react-router lib, on browsers it's a good practice to have that on the dep list, I'm not sure if we could have an loose of sync on the electron context, wdyt?

https://github.com/remix-run/react-router/blob/main/packages/react-router/lib/hooks.tsx

![image](https://user-images.githubusercontent.com/50559336/183931623-229fa318-fe74-4b4c-b5ce-0e184de998cf.png)

Reply:
I think we can remove it and if we got any problem in that context we can add it again. less is more.